internal/web/results.go

```diff
@@ -185,6 +211,50 @@ func renderInProgress(w http.ResponseWriter, r *http.Request, badge []byte, vali
 	}
 }
 
+func myReadDir(dirname string) ([]os.FileInfo, error) {
+	f, err := os.Open(dirname)
+	if err != nil {
+		return nil, err
+	}
+	list, err := f.Readdir(-1)
+	f.Close()
+	if err != nil {
+		return nil, err
+	}
+	sort.Slice(list, func(i, j int) bool { return list[i].ModTime().Unix() > list[j].ModTime().Unix() })
+	return list, nil
+}
+
+func resultsHistory(validator, user, repo string) ResultsHistoryStruct {
+	var ret ResultsHistoryStruct
+	srvcfg := config.Read()
+	resdir := filepath.Join(srvcfg.Dir.Result, validator, user, repo)
+	fileinfos, err := myReadDir(resdir)
+	if err != nil {
+		log.ShowWrite("[Error] cannot retrieve results history '%s/%s' result: %s\n", user, repo, err.Error())
+		return ret
+	}
+	for _, i := range fileinfos {
+		pth := filepath.Join("/results", strings.Split(resdir, "results/")[1], i.Name())
+		fp := filepath.Join(resdir, i.Name(), srvcfg.Label.ResultsBadge)
+		badge, err := ioutil.ReadFile(fp)
+		if err != nil {
+			badge = []byte("<svg></svg>")
+			fmt.Printf("%+v\n", err)
+		}
+		if i.IsDir() && i.Name() != "." {
+			var res Result
+			res.Href = pth
+			res.Alt = i.Name()
+			res.Text1 = i.ModTime().Format("2006-01-02")
+			res.Text2 = i.ModTime().Format("15:04:05 -07:00")
```

Comment:
Please use time format "hh:mm" only instead of the full time stamp.

Reply:
The time will be in GMT then, hopefully it will not cause some confusion to the user.